docs/docs/intro.md

```diff
@@ -13,25 +13,31 @@ feature.
 
 ## Features
 
-At this point, ZMK is _missing_ more features than it has. Currently, the mostly working bits
+At this point, ZMK is _missing_ more features than it has. Currently, the working bits
+At this point, ZMK is still missing many features. Currently, the working bits
 include:
 
-- HID Over GATT (HOG) - This is the official term for BLE HID devices
-- Keymaps and layers with basic keycodes
-- Some initial work on one "behavior", Mod-Tap
-- Basic HID over USB
-- Basic consumer (media) keycodes.
-- Basic OLED display logic
-- Basic Split support
-- Encoders
+- Bluetooth connectivity (HID Over GATT (HOG) - This is the official term for BLE HID devices)
+- USB connectivity
+- Low active power usage
+- Split keyboard support
+- [Keymaps and layers](behavior/layers.md)
+- [Hold-tap](behavior/hold-tap.md) (which includes [mod-tap](behavior/mod-tap.md), [layer-tap](behavior/layers.md))
+- [Basic HID over USB](behavior/key-press.md)
+- [Basic consumer (media) keycodes](behavior/key-press.md)
```

Comment:
You should be able to link to the generated *anchor* to get right to this section, e.g.:

```suggestion
- [Basic consumer (media) keycodes](/docs/behavior/key-press#consumer-key-press)
```